### PR TITLE
Switch to using private hosted pool for Project Reunion Foundation

### PR DESCRIPTION
**Pull request by kythant**
Switch to using private hosted pool for Project Reunion Foundation

### CODE REVIEW

File: build/ProjectReunion-BuildFoundation.yml [hunk 1]
```diff
@@ -13,8 +13,7 @@ parameters:
 
 jobs:
 - job: CredScan
-  pool:
-    vmImage: 'windows-2019'
+  pool: $(ProjectReunionBuildPool)
```
Comment (huichen123) [inline build/ProjectReunion-BuildFoundation.yml:16]: >ProjectReunionBuildPool [](http://example.com/codeflow?start=10&length=23)

do we need to have a default value in yml?

Comment (kythant) [reply build/ProjectReunion-BuildFoundation.yml:16]: I don't think we need to have one.

**Comment by huichen123** [comment]
>     name: Package ES Lab E

are they going to decommission those Package ES lab very soon?

---
Refers to: build/ProjectReunion-BuildFoundation.yml:187 in 992ffc4. [](commit_id = 992ffc42184f41947a5b3e9a7d79ae8d2a852bb2, deletion_comment = False)

**Review by huichen123 — APPROVED** [review]
:shipit:

**Pull request merged**